core/src/worker/workflow/machines/nexus_operation_state_machine.rs

```diff
@@ -65,10 +68,17 @@ fsm! {
     Started
       --(NexusOperationTimedOut(NexusOperationTimedOutEventAttributes), on_timed_out)--> TimedOut;
 
+    Cancelled --(Cancel)--> Cancelled;
+    Cancelled --(CommandRequestCancelNexusOperation)--> Cancelled;
+    Cancelled --(NexusOperationCancelRequested)--> Cancelled;
+    Cancelled --(NexusOperationCompleted(NexusOperationCompletedEventAttributes), shared on_completed)--> Cancelled;
+    Cancelled --(NexusOperationFailed(NexusOperationFailedEventAttributes), shared on_failed)--> Cancelled;
+    Cancelled --(NexusOperationTimedOut(NexusOperationTimedOutEventAttributes), shared on_timed_out)--> Cancelled;
+    Cancelled --(NexusOperationCanceled(NexusOperationCanceledEventAttributes))--> Cancelled;
+
     // Ignore cancels in all terminal states
     Completed --(Cancel)--> Completed;
     Failed --(Cancel)--> Failed;
-    Cancelled --(Cancel)--> Cancelled;
```

Comment:
I'm a little worried removing this could cause errors in SDKs that allow invoking a cancel on an old handle or something. Probably no harm in leaving it in?

Reply:
This guy is still gone